<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editwindow</class>
 <widget class="QDialog" name="editwindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>245</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/resources.qrc">
    <normaloff>:/xtodo.png</normaloff>:/xtodo.png</iconset>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="priorityLabel">
     <property name="font">
      <font>
       <family>Noto Sans</family>
      </font>
     </property>
     <property name="text">
      <string>Priority:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="priorEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="complDateLabel">
     <property name="font">
      <font>
       <family>Noto Sans</family>
      </font>
     </property>
     <property name="text">
      <string>Completion date:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="complDateEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="creatDateLabel">
     <property name="font">
      <font>
       <family>Noto Sans</family>
      </font>
     </property>
     <property name="text">
      <string>Creation date:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="createDateEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="textLabel">
     <property name="font">
      <font>
       <family>Noto Sans</family>
      </font>
     </property>
     <property name="text">
      <string>Text:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="textEdit"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="contextLabel">
     <property name="font">
      <font>
       <family>Noto Sans</family>
      </font>
     </property>
     <property name="text">
      <string>Context tag:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="contextEdit"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="projectLabel">
     <property name="font">
      <font>
       <family>Noto Sans</family>
      </font>
     </property>
     <property name="text">
      <string>Project tag:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="projectEdit"/>
   </item>
   <item row="6" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
       </property>
       <property name="centerButtons">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>editwindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>editwindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
